From: Simon McVittie <smcv@collabora.com>
Date: Tue, 31 Oct 2017 15:20:49 +0000
Subject: userns-block-fd: Search $PATH for python

On systems without the /usr merge, it's almost certainly in /usr,
so this script would have failed.

Signed-off-by: Simon McVittie <smcv@collabora.com>

Closes: #246
Approved by: giuseppe
Applied-upstream: 0.2.1, commit:30338c3f0db758379e87277c08b4aa66168e1a26
---
 demos/userns-block-fd.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/demos/userns-block-fd.py b/demos/userns-block-fd.py
index 0677a0d..a390cce 100755
--- a/demos/userns-block-fd.py
+++ b/demos/userns-block-fd.py
@@ -1,4 +1,4 @@
-#!/bin/python
+#!/usr/bin/env python
 
 import os, select, subprocess, json
 
